src/main/resources/project/parameterForms/ec_parameterForm-StartCluster.xml

```diff
@@ -0,0 +1,46 @@
+<!--
+
+     Copyright 2015 Electric Cloud, Inc.
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+-->
+<editor>
+  <help>/commander/pages/@PLUGIN_NAME@/@PLUGIN_KEY@_help?s=Administration&amp;ss=Plugins#StartCluster</help>
+
+  <formElement>
+    <type>entry</type>
+    <label>WLST Absolute Path:</label>
+    <property>wlstabspath</property>
+    <required>1</required>
+    <documentation>Absolute path to WLST. i.e: '/path/to/wlst.cmd' or 'wlst.sh'.</documentation>
```

Comment:
I guess the parameter name should be called "Path to wlst" and there should be no word 'absolute' in documentation, since we can work with PATH.